install.sh

```diff
@@ -9,6 +13,7 @@ needed_package_name=null_package_name
 stage=release
 interactive=1
 skip_install=0
+skip_checks=0
```

Comment:
The idea was that if/when we have future checks (like /tmp writable, etc.) they would all be covered by this one flag. Do you think --skip-checks should also skip the running collect check?